conjur/argument_parser/_list_parser.py

```diff
@@ -50,7 +50,7 @@ def _create_list_parser(self):
                             '    conjur list --kind group --members-of conjur-root-admins\t\t'
                             'List members within a role passing the role '
                             'type in the --kind option\n'
-                            '    conjur list --kind group --permitted-members-of '
+                            '    conjur list --kind group --permitted-roles '
                             'conjur-root-admins --privilege read\t\t'
                             'Lists the roles which have the named permission on a resource.\n'
```

Comment:
@szamir1 please supply updated text since `--kind` is not supported like that, only as a filter.